src/components/toast/toast.spec.tsx

```diff
@@ -593,6 +593,53 @@ describe("Align", () => {
   );
 });
 
+describe("Align vertical", () => {
```

Comment:
Comment: we could probably add some tests that capture the different permutations of the alignment props, but maybe that's better suited to cypress or chromatic as that's rendered in browser